TeX_files/chapter04.tex

```diff
@@ -130,7 +130,7 @@ \section{Impedance Transform Relationship}
 The impedance at any point $l$ is related to the load impedance $Z_L$ and the characteristics impedance $Z_0$ by the above relationship. This enables us to move $Z_L$ to any point along the transmission line. Hence at $l = 0$, we have $Z_L$ but at $l = L$, the input impedance measured at the generator end will not be $Z_L$. It will depend on $Z_L$ and also the length of the line. So, if the length of the line keeps varying, the impedance you measure at the input end of the line will keep varying. 
 
 Hence, if we design a circuit at high frequency and connect a load to the end, depending on the length of the transmission line, the input impedance will keep varying with length. From a circuit perspective, the input impedance is very important. So at high frequencies, the length connecting the load to the circuit matters as the input impedance seen by the generator varies with length.\\\\
-From Equation~\ref{eqn:imp}:
+From Equation \ref{eqn:imp}:
```

Comment:
@Victor-Aken `~\ref{}` and ` \ref{}` work the same way. Just needed to let you know